src/services/widgets/widgets.svc.coffee

```diff
@@ -189,30 +190,36 @@ angular
           authHeader = 'Basic ' + btoa(_self.getSsoSessionId())
           config = { headers: {'Authorization': authHeader } }
 
-          $http.get(url, config).then(
-            (success) ->
-              content = success.data.content || success.data[widget.endpoint] || {}
-              if _.isEmpty(content)
-                # Reload the widget with param `demo` set at `true` (to retrieve stub data)
-                if demoData
-                  # If we were already trying to load the widget in demo mode, we resolve the widget without content
-                  # TODO: display an error box?
-                  $log.error('Impac! - WidgetsSvc: Cannot retrieve demo data for widget:', widget)
-                  $q.resolve(widget)
+          $http.get(url, config)
+            .then(
+              (success) ->
+                ImpacDateFormatter.formatDateString success.data, widget.endpoint
```

Comment:
Same comment as per dashboards svc: this is too intrusive for the existing code. Please apply the formatting in the view directly